kanidm_client/src/lib.rs

```diff
@@ -208,6 +208,12 @@ impl KanidmClientBuilder {
         config_path: P,
     ) -> Result<Self, ()> {
         debug!("Attempting to load configuration from {:#?}", &config_path);
+
```

Comment:
"Config file not found" is already covered below this?

Reply:
Ahahah there was a special-hell error here where when the file doesn't exist, docker mounts were passing it through as a directory and it caused some ridiculous errors like "can't read file, this is a directory". that's why I added the extra check. I might comment that in the code.

Reply:
Ah, I *hate* that "feature" in container-land.